root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -137,6 +137,12 @@ const testData = [
              input_entity_type: 'release',
     expected_relationship_type: 'downloadpurchase',
   },
+  {
+                     input_url: 'https://us.7digital.com/yourmusic/artist/falco/release/vienna-greatest-hits-311837/311837',
+             input_entity_type: 'release',
+    expected_relationship_type: 'downloadpurchase',
+            expected_clean_url: 'https://us.7digital.com/artist/falco/release/vienna-greatest-hits-311837/311837',
```

Comment:
Hmm, just removing `/yourmusic` doesn't fix the link though. The edit in the ticket also had to remove the `/311837` at the end (otherwise it's a 404). Not sure if those are a standard part of the "yourmusic" URLs or what.

Reply:
Oh, whoops. I'll ask.

Reply:
Updated the regex for this.